dashboard/app/controllers/api/v1/json_api_controller.rb

```diff
@@ -5,4 +5,9 @@ class Api::V1::JsonApiController < ApplicationController
   rescue_from CanCan::AccessDenied do
     head :forbidden
   end
+
+  # Don't differentiate between missing and forbidden records
+  rescue_from ActiveRecord::RecordNotFound do
+    head :forbidden
+  end
```

Comment:
For some reason I was thinking it's good practice to return uniform error messages so an attacker can't crawl for valid section codes / ids - but that's probably more of a username thing, huh?  I guess this isn't really needed.